### PR TITLE
rock paper scissors post DOM manipulation merge

### PR DESCRIPTION
**Pull request by harrika**
This is a PR template. If you are adding a solution link to the curriculum, leave this as is. If not, delete it and write the message you wish.

Thank you,

The Odin Project team.

### CODE REVIEW

**Comment by harrika** [comment]
request pull for project solutions

File: web_development_101/javascript_basics/project_rock_paper_scissors.md [hunk 1]
```diff
@@ -41,12 +41,17 @@ Submit a solution with a pull request to this [file](https://github.com/TheOdinP
 
 - Add your solution below this line!
 - [CoderOO7 solution (with GUI)](https://github.com/coderoo7/rockPaperScissor) - [View in Browser](https://coderoo7.github.io/rockPaperScissor)
+>>>>>>> 5e1cfee47336aeec36ff2c68e43c5b5f1d0a44e9
```
Comment (KevinMulhern) [inline web_development_101/javascript_basics/project_rock_paper_scissors.md]: It looks like these merge conflict markers were left behind, can you remove them please? any line that starts with`>>>>` will be merge conflict markers.

Comment (harrika) [reply web_development_101/javascript_basics/project_rock_paper_scissors.md]: removed markers thanks

**Comment by harrika** [comment]
change effected

**Pull request merged**